packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -661,7 +667,7 @@ export default class SalesforceCRMService implements CRM {
         if (foundFields.length === fieldSet.size) break;
 
         if (fieldSet.has(field.name)) {
-          foundFields.push(field.name);
+          foundFields.push(field);
```

Comment:
Instead of returning just the field names, we're returning the whole field object.